.microcks/overlays.yaml

```diff
@@ -0,0 +1,204 @@
+    remove: true
+
+  - target: $.paths["/bookings"].get.responses["200"].content["application/json"]
+    update:
+      examples:
+        bookings:
+          summary: A list of bookings
+          value:
+            data:
+              - id: bfc5af2c-f477-43c4-8bdf-a00bdb939d65
+                trip_id: efdbb9d1-02c2-4bc3-afb7-6788d8782b1e
+                passenger_name: John Doe
+                has_bicycle: true
+                has_dog: true
+              - id: 1725ff48-ab45-4bb5-9d02-88745177dedb
+                trip_id: b2e783e1-c824-4d63-b37a-d8d698862f1d
+                passenger_name: Jane Smith
+                has_bicycle: false
+                has_dog: false
+            links:
+              self: https://api.example.com/bookings
+              next: https://api.example.com/bookings?page=2
+
+  - target: $.paths["/bookings"]["post"].responses["201"].content["application/json"]["example"]
+    remove: true
+
+  - target: $.paths["/bookings"]["post"].responses["201"].content["application/json"]
+    update:
+      examples:
+        new_booking:
+          summary: New Booking
+          value: |-
+            {
+              "id": "{{ uuid() > put(bookingId) }}",
+              "trip_id": "{{ request.body/trip_id }}",
+              "passenger_name": "{{ request.body/passenger_name }}",
+              "has_bicycle": {{ request.body/has_bicycle }},
+              "has_dog": {{ request.body/has_dog }},
+              "links": {
+                "self": "https://api.example.com/bookings/{{ bookingId }}"
+              }
+            }
+
+  - target: $.paths["/bookings/{bookingId}"].parameters[?(@.name=="bookingId")].example
+    remove: true
+    
+  - target: $.paths["/bookings/{bookingId}"].parameters[?(@.name=="bookingId")]
+    update:
+      examples:
+        booking_1725ff48-ab45-4bb5-9d02-88745177dedb:
+          value: '1725ff48-ab45-4bb5-9d02-88745177dedb'
+        booking_bfc5af2c-f477-43c4-8bdf-a00bdb939d65:
+          value: 'bfc5af2c-f477-43c4-8bdf-a00bdb939d65'
+
+  - target: $.paths["/bookings/{bookingId}"].get.responses["200"].content["application/json"]["example"]
+    remove: true
+
+  - target: $.paths["/bookings/{bookingId}"].get.responses["200"].content["application/json"]
+    update:
+      examples:
+        booking_1725ff48-ab45-4bb5-9d02-88745177dedb:
+          summary: John Doe
+          value:
+            id: 1725ff48-ab45-4bb5-9d02-88745177dedb
+            trip_id: efdbb9d1-02c2-4bc3-afb7-6788d8782b1e
+            passenger_name: John Doe
+            has_bicycle: true
+            has_dog: true
+            links:
+              self: https://api.example.com/bookings/1725ff48-ab45-4bb5-9d02-88745177dedb
+        booking_bfc5af2c-f477-43c4-8bdf-a00bdb939d65:
+          summary: Billy Bikeless
+          value:
+            id: bfc5af2c-f477-43c4-8bdf-a00bdb939d65
+            trip_id: efdbb9d1-02c2-4bc3-afb7-6788d8782b1e
```

Comment:
and last a trip id from the examples data `ea399ba1-6d95-433f-92d1-83f67b775594`?